examples/complex-chart/webpack.config.js

```diff
@@ -0,0 +1,48 @@
+// NOTE: This is a Webpack 2 configuration file for react-vis
+const {resolve} = require('path');
+
+module.exports = {
+  entry: {
+    app: resolve('./app.js')
+  },
+
+  devServer: {
+    stats: {
+      warnings: false
+    }
+  },
+
+  devtool: 'source-maps',
+
+  // NOTE: If you are using this config for your app, do not use this 'resolve'
+  //       section. This is example specific
+  resolve: {
+    alias: {
+      'react-vis': resolve('../../dist'),
```

Comment:
By removing the `react-vis` alias to `../../dist` dir, the examples will be built against the latest reactt-vis npm package instead of the source. Is that okay?